gravitee-node-kubernetes/src/main/java/io/gravitee/node/kubernetes/keystoreloader/AbstractKubernetesKeyStoreLoader.java

```diff
@@ -41,7 +41,7 @@
  */
 public abstract class AbstractKubernetesKeyStoreLoader<T> implements KeyStoreLoader {
 
-    private static final Logger logger = LoggerFactory.getLogger(AbstractKubernetesKeyStoreLoader.class);
+    protected static final Logger logger = LoggerFactory.getLogger(AbstractKubernetesKeyStoreLoader.class);
```

Comment:
each class deserve it's own logger imo

Reply:
This was just a mistake, I'll revert it back